test/package.mjs

```diff
@@ -128,36 +124,42 @@ test('Every rule is defined in readme.md usage and list of rules in alphabetical
 
 	t.truthy(usageRules, 'List of rules should be defined in readme.md ## Usage and be valid JSON');
 
-	const rulesMatch = /<!-- RULES_TABLE_START -->(?<rulesText>.*?)<!-- RULES_TABLE_END -->/ms.exec(readme);
-	t.truthy(rulesMatch, 'List of rules should be defined in readme.md in ## Rules before ## Recommended config');
-	const {rulesText} = rulesMatch.groups;
-	const re = /\| \[(?<id>.*?)]\((?<link>.*?)\) \| (?<description>.*) \| (?<recommended>.*?) \| (?<fixable>.*?) \| (?<hasSuggestions>.*?)\n/gm;
-	const rules = [];
-	let match;
-	do {
-		match = re.exec(rulesText);
-		if (match) {
-			const {id, link, description} = match.groups;
-			t.is(link, `docs/rules/${id}.md`, `${id} link to docs should be correct`);
-			t.true(description.trim().length > 0, `${id} should have description in readme.md ## Rules`);
-			rules.push(id);
-		}
-	} while (match);
```

Comment:
OK to remove this since it was quite ugly, but I guess we have no sanity check that the `generate-rules-table` script works correctly anymore.

Reply:
Restored https://github.com/sindresorhus/eslint-plugin-unicorn/pull/1682/commits/a4dc5d8004ee55eea4be14122480ddd75688aa3f